contrib/src/main/java/com/datatorrent/contrib/parser/DelimitedSchema.java

```diff
@@ -268,65 +220,16 @@ public String toString()
    * field has a name, type and a set of associated constraints.
    *
    */
-  public class Field
+  public class DelimitedField extends Schema.Field
   {
     /**
-     * name of the field
-     */
-    String name;
-    /**
-     * Data type of the field
-     */
-    FieldType type;
-    /**
      * constraints associated with the field
      */
     Map<String, Object> constraints = new HashMap<String, Object>();
 
-    public Field(String name, String type)
-    {
-      this.name = name;
-      this.type = FieldType.valueOf(type.toUpperCase());
-    }
-
-    /**
-     * Get the name of the field
-     *
-     * @return name
-     */
-    public String getName()
-    {
-      return name;
-    }
-
-    /**
-     * Set the name of the field
-     *
-     * @param name
-     */
-    public void setName(String name)
-    {
-      this.name = name;
-    }
-
-    /**
-     * Get {@link FieldType}
-     *
-     * @return type
-     */
-    public FieldType getType()
-    {
-      return type;
-    }
-
-    /**
-     * Set {@link FieldType}
-     *
-     * @param type
-     */
-    public void setType(FieldType type)
+    public DelimitedField(String name, String type)
```

Comment:
This is probably a backward incompatible change. The class DelimitedSchema.Field would not exist anymore...